honeybee_energy/properties.py

```diff
@@ -21,18 +23,16 @@ def construction(self):
 
         If construction is not set by user then it will be assigned based on parent zone
         construction-set when the face is added to a zone. For a free floating face the
-        construction will be set to a generic construction based on face type and face
-        boundary condition.
+        construction will be None unless it is set by user.
         """
         if self._construction:
             # set by user
             return self._construction
         elif self._constructionset:
             # set by parent zone
-            return self._constructionset(self.face_type, self.boundary_condition)
+            return self._constructionset(self._face_type, self._boundary_condition)
         else:
-            # not set yet - use generic construction set
-            return generic_constructionset(self.face_type, self.boundary_condition)
+            return None
```

Comment:
Why have we gotten rid of the generic construction set here?

Reply:
Because it was working against us in `to_dict` conversion and will work against us in creating a Model. In both cases passing None is much simpler to evaluate. We know that we must use the Default Construction Set however if we had some default value then we have to do an extra check to see if it is the default construction for this type.

I know it will be confusing in GH interface so we have to find a better solution there to expose default values but as we discussed we will optimize the core libraries for Pollination and find workarounds for plugins.

Reply:
I see. That makes sense and I agree.